packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -662,6 +662,41 @@ class _FloatingActionButtonTransitionState extends State<_FloatingActionButtonTr
 /// [ScaffoldState] for the current [BuildContext] via [Scaffold.of] and use the
 /// [ScaffoldState.showSnackBar] and [ScaffoldState.showBottomSheet] functions.
 ///
+/// ## Sample code
+///
+/// This example shows a [Scaffold] with an [AppBar], a [BottomAppBar]
+/// and a [FloatingActionButton]. The [body] is a [Text] placed in a [Center]
+/// in order to center the text within the [Scaffold] and the
+/// [FloatingActionButton] is centered and docked within the
+/// [BottomAppBar] using [FloatingActionButtonLocation.centerDocked].
+///
+/// ```dart
+/// Scaffold(
+///      appBar: AppBar(
+///        title: Text('Sample Code'),
+///      ),
+///      body: Center(
+///        child: Text(
+///          'Scaffold',
+///        ),
+///      ),
+///
+///      bottomNavigationBar: BottomAppBar(
```

Comment:
nit: we should be consistent about whether we want blank lines between each entry in this list or not. I'd probably go with no blank lines to be consistent with other sample code, but I can go either way if you think it's important for this example.